libs/menu/src/TopNavigation.tsx

```diff
@@ -514,7 +543,7 @@ export function TopNavigationDropdown({
           <div className="px-2">{children}</div>
         </DropdownMenu>
       </div>
-      <div className={`md:hidden flex flex-col ${opened && "w-80"}`}>
+      <div className={`md:hidden flex flex-col ${opened ? "w-full" : "pl-14"}`}>
```

Comment:
Is pl-14 actually necessary?

Reply:
Yes, otherwise the button stretches too much on mobile display. This way it has normal width and looks the same as SidebarNavigation's equivalent on mobile devices.

Reply:
![image](https://user-images.githubusercontent.com/69048022/216619078-6cacd97a-f273-45ae-a3f2-ddfd85c305e7.png)
It seems strange and different from SidebarNavigation.